spec/lib/listen/backend_spec.rb

```diff
@@ -33,33 +33,33 @@
     stub_const('Listen::Adapter::Config', adapter_config_class)
 
     allow(adapter_config_class).to receive(:new).
-      with([dir1], queue, silencer, adapter_options).
```

Comment:
The current indentation looks better to me. This is one of the side effects as line length offence gets into action sometimes while indenting.

Reply:
Same comment.

Reply:
This should work.
```ruby
Layout/MultilineMethodCallIndentation:
  EnforcedStyle: indented
```

Reply:
Changed.